builtin/providers/vcd/resource_vcd_vapp.go

```diff
@@ -207,6 +211,23 @@ func resourceVcdVAppUpdate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Error getting VApp status: %#v", err)
 	}
 
+	if d.HasChange("guest_properties") {
+		_, nraw := d.GetChange("guest_properties")
+		tmp := nraw.(map[string]interface{})
```

Comment:
Oops, I don't need a temporary variable here; I can update 218 to check range nraw.(map[string]interface{}) instead.